react/IntentModal/IntentModal.jsx

```diff
@@ -1,44 +1,64 @@
-import React from 'react'
-import Modal from '../Modal'
-import IntentIframe from '../IntentIframe'
+import React, { Component } from 'react'
 import styles from './styles.styl'
 import PropTypes from 'prop-types'
 
+import { once } from 'lodash'
```

Comment:
Could you, at least, use directly `lodash/once` here? We should install only lodash helpers necessary in the library and not globalize the lodash whole usage bin adding the `lodash` whole package IMHO.